sqlalchemy_redshift/dialect.py

```diff
@@ -744,63 +747,74 @@ def _get_all_relation_info(self, connection, **kw):
             relations[key] = rel
         return relations
 
+    # We fetch column info an entire schema at a time to improve performance
+    # when reflecting schema for multiple tables at once.
     @reflection.cache
-    def _get_all_column_info(self, connection, **kw):
+    def _get_schema_column_info(
+        self, connection, schema=None, **kw
+    ):
+        where_schema = "WHERE schema = :schema" if schema else ""
+        and_schema = "AND view_schema = :schema" if schema else ""
         all_columns = defaultdict(list)
         with connection.connect() as cc:
-            result = cc.execute("""
```

Comment:
Please try to maintain the existing indentation unless there's a significant structural change to the query. It's currently difficult to tell what's changing in this diff.

Reply:
Yeah, apologies, I've reverted the query back to its formatting in master.